hack/packages/package-tools/cmd/package-bundle-generate.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/vmware-tanzu/tanzu-framework/hack/packages/package-tools/constants"
+
+	"github.com/spf13/cobra"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/vmware-tanzu/carvel-imgpkg/pkg/imgpkg/lockconfig"
+	"github.com/vmware-tanzu/tanzu-framework/hack/packages/package-tools/utils"
+)
+
+var packageRepository, version, subVersion string
+var all bool
+
+// packageBundleGenerateCmd is for generating package bundle
+var packageBundleGenerateCmd = &cobra.Command{
```

Comment:
Is there a reason to make this a cobra.Command if its use is only going to be to `go run` it from our Makefile?